src/content/docs/en/guides/cms/frontmatter-cms.mdx

```diff
@@ -0,0 +1,124 @@
+
+## Integration with Astro
+
+In this section, we'll walk through how to add Front Matter CMS to your Astro project.
+
+### Prerequisites 
+
+- Visual Studio Code
+- Project with content, a great start is the [Astro Blog template](https://github.com/withastro/astro/tree/main/examples/blog)
+
+### Install the Front Matter CMS extension
+
+You can get the extension from the [Visual Studio Code Marketplace - Front Matter](https://marketplace.visualstudio.com/items?itemName=eliostruyf.vscode-front-matter) or by clicking on the following link: [open Front Matter CMS extension in VS Code](vscode:extension/eliostruyf.vscode-front-matter)
+
+### Project initialization
+
+Once Front Matter CMS is installed, you will get a new icon in the Activity Bar on the left side (or right side depending on your preference) of the editor. Click on it to open the **Front Matter CMS** panel.
+
+- Click on the **Initialize project** button;
+- The welcome screen will open and you can start initializing the project;
+- Click on the first step to **Initialize project**;
+- As Astro is one of the supported frameworks, you can select it from the list;
+- Register your content folders, in this case, the `src/content/blog` folder;
+- Click on **Show the dashboard** to open the content dashboard.
+
+### Project configuration
+
+Once the project is initialized, you will get a `frontmatter.json` configuration file and a `.frontmatter` folder in the root of your project.
+
+<FileTree title="Project Structure">
+- .frontmatter/
+  - database/
+    - mediaDb.json
+- src/
+- astro.config.mjs
+- **frontmatter.json**
+- package.json
+</FileTree>
+
+#### Content-type configuration
+
+Content-types is the way Front Matter CMS knows how to handle your content. You can configure the content-types in the `frontmatter.json` file. 
+
+- Open the `frontmatter.json` file;
+- Replace the `frontMatter.taxonomy.contentTypes` array with the following content-types configuration:
+
+```json title="frontmatter.json"
+{
+  "frontMatter.taxonomy.contentTypes": [
+    {
+      "name": "default",
+      "pageBundle": false,
+      "previewPath": "'blog'",
+      "filePrefix": null,
+      "fields": [
+        {
+          "title": "Title",
+          "name": "title",
+          "type": "string",
+          "single": true
+        },
+        {
+          "title": "Description",
+          "name": "description",
+          "type": "string"
+        },
+        {
+          "title": "Publishing date",
+          "name": "pubDate",
+          "type": "datetime",
+          "default": "{{now}}",
+          "isPublishDate": true
+        },
+        {
+          "title": "Content preview",
+          "name": "heroImage",
+          "type": "image",
+          "isPreviewImage": true
+        }
+      ]
+    }
+  ]
+}
+```
+
+:::note
+This configuration makes sure that the content-type matches the blog content in its fields.
```

Comment:
Just to be clear, by "the blog content", do you mean the content in the blog template?

If so, the suggestion to use the Astro blog template earlier needs to be a bit of a harder recommendation.